applications/plugins/Compliant/mapping/AdditionMapping.h

```diff
@@ -121,7 +121,7 @@ class SOFA_Compliant_API AdditionMapping : public ConstantAssembledMapping<TIn,
     void draw(const core::visual::VisualParams* vparams) override
     {
 
-#ifndef SOFA_NO_OPENGL
+#ifdef COMPLIANT_HAVE_SOFA_GL
```

Comment:
hummmm, should not it be #if COMPLIANT_HAVE_SOFA_GL ==1 ? 🤠

Reply:
(and same for the others)

Reply:
Haha you god damn right

Reply:
Is the `== 1` really needed though?

Reply:
dont think so actually, just the if should be enough indeed